osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -272,7 +272,7 @@ private void load(AudioManager audio, DialogOverlay dialog, OsuColour colours, S
             if (Footer != null)
             {
                 Footer.AddButton(new FooterButtonMods { Current = Mods }, ModSelect);
-                Footer.AddButton(new FooterButtonRandom { Action = triggerRandom });
+                Footer.AddButton(new FooterButtonRandom { PrimaryAction = triggerNextRandom, SecondaryAction = triggerPreviousRandom });
```

Comment:
```suggestion
                Footer.AddButton(new FooterButtonRandom { 
                    PrimaryAction = Carousel.SelectNextRandom, 
                    SecondaryAction = Carousel.SelectPreviousRandom
                });
```

No need for the private redirect functions below.

Reply:
Carousel.SelectNextRandom returns a bool which throws a compiler error?

Reply:
`() => Carousel.SelectNextRandom` in that case